### PR TITLE
[Component with its own class name as service type] LPS-198099 Refactor UserPersonalSitePermissions component

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-198099

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:sf

File: modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/model/listener/CompanyModelListener.java [hunk 1]
```diff
@@ -67,6 +108,219 @@ public void onAfterCreate(Company company) {
...
+			ServiceReference<PanelApp> serviceReference, PanelApp panelApp) {
+
+			_bundleContext.ungetService(serviceReference);
+		}
+
+		private final BundleContext _bundleContext;
+
+	}
+
+	private class UserPersonalSitePermissions {
```
Comment (rafaprax) [inline modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/model/listener/CompanyModelListener.java:178]: you do not need this inner class... just merge the `UserPersonalSitePermissions` methods into the `CompanyModelListener` sounds good.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by MrJohn1911** [comment]
Pull Request forwarded: https://github.com/liferay-core-infra/liferay-portal/pull/5778

**Pull request closed without merging**